.github/workflows/python-versions.yml

```diff
@@ -3,8 +3,8 @@ name: Python Versions
 on:
   push:
     branches:
-      - 'develop'
-      - 'feature/**'
```

Comment:
All of the changes that aren't adding typechecking to the `py_snowflake` below were done by the autoformatter

Reply:
I'll probably also make a second PR to make these changes straight to `develop` to shrink the diff for the feature branch tomorrow